docs/how-to/rocks/use-flask-extension.rst

```diff
@@ -28,15 +56,90 @@ be prefixed with ``flask/app``.
 
 For example, to include only select files:
 
-.. literalinclude:: ../code/use-flask-extension/prime_example/rockcraft.yaml
+.. literalinclude:: ../code/use-flask-extension/prime-example/rockcraft.yaml
     :language: yaml
     :start-after: [docs:parts-start]
     :end-before: [docs:parts-end]
 
-To exclude certain files from the project directory in the rock image,
-add the following part to ``rockcraft.yaml``:
+To exclude certain files from the project directory in the rock image and
+include everything else, add the following part to ``rockcraft.yaml``:
 
-.. literalinclude:: ../code/use-flask-extension/prime_exclude_example/rockcraft.yaml
+.. literalinclude:: ../code/use-flask-extension/prime-exclude-example/rockcraft.yaml
     :language: yaml
     :start-after: [docs:parts-start]
     :end-before: [docs:parts-end]
+
+Including additional debs in the OCI image
+------------------------------------------
+
+if your app requires debs, for example to connect to a database, add the
```

Comment:
Nit picks: Capitalise "if". I would also restructure slightly to say "If your app requires debs (for example, to connect to a database), add..."

Reply:
Fixed